.env.development

```diff
@@ -122,3 +122,11 @@ SUPERVISOR_EMAILS='Supervisor <supervisor@test.com>'
 SEED_GENERATED_ADMIN_PASSWORD='Resolve2SaveLives'
 SEED_GENERATED_ACTIVE_USER_ROLE="Seed User | Active"
 SEED_GENERATED_INACTIVE_USER_ROLE="Seed User | Inactive"
+
+#
+# DHIS2 credentials
+#
+DHIS2_URL=
+DHIS2_USERNAME=
+DHIS2_PASSWORD=
+DHIS2_VERSION="2.28"  # This is a hack because the dhis2 gem only goes to 2.28
```

Comment:
Not sure I understand this comment...because the gem we have locked is 3.3.1.  Can you expand on this for folks who aren't super familiar w/ how the gem and server interact?

Reply:
This is the DHIS2 version installed on the server. The API is apparently tied to specific versions (?!)

We will need to update the gem in the future.

Reply:
```suggestion
DHIS2_VERSION="2.28"  # This is a hack because the current dhis2 gem only works against version 2.28 of the DHIS API
```